SmartDeviceLink-iOS.podspec

```diff
@@ -24,8 +24,12 @@ ss.public_header_files = [
 'SmartDeviceLink/SDLProxyListener.h',
 'SmartDeviceLink/SDLSecurityType.h',
 'SmartDeviceLink/SDLStreamingMediaManager.h',
+'SmartDeviceLink/SDLTouch.h',
 'SmartDeviceLink/SDLTouchManager.h',
 'SmartDeviceLink/SDLTouchManagerDelegate.h',
+'SmartDeviceLink/SDLConsoleController.h',
+'SmartDeviceLink/SDLDebugTool.h',
+'SmartDeviceLink/SDLDebugToolConsole.h',
```

Comment:
These don't exist on this branch. See SmartDeviceLink.h for all the files that should be in here.